examples/example_plugin/mkdocs.yml

```diff
@@ -0,0 +1,23 @@
+dev_addr: '127.0.0.1:8001'
+site_dir: example_plugin/static/example_plugin/docs
+site_name: Example Plugin Documentation
+python:
+    install:
+        - requirements: docs/requirements.txt
+theme:
+    name: readthedocs
+    navigation_depth: 4
+    hljs_languages:
+        - django
+        - yaml
+extra_css:
+    - extra.css
```

Comment:
_Is_ there an `extra.css` for the plugin docs? I don't see it in these diffs.

Reply:
I fixed this, but probably forgot to commit it.

Reply:
Resolved.